src/test/scala/com/databricks/spark/xml/XmlSuite.scala

```diff
@@ -17,18 +17,17 @@ package com.databricks.spark.xml
 
 import java.io.File
 import java.nio.charset.UnsupportedCharsetException
+import java.nio.file.Files
 import java.sql.{Date, Timestamp}
 
 import scala.io.Source
-
```

Comment:
Let's keep the import order - https://github.com/databricks/scala-style-guide#imports

Reply:
Fixed, updated my IDE to follow guidelines